src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,57 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<I, S> {
+    private I first;
+    private S second;
 
+    private Pair(I first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public static <I, S> Pair of(I first, S second) {
+        return new Pair(first, second);
```

Comment:
```suggestion
    public static <I, S> Pair<I, S> of(I first, S second) {
        return new Pair<>(first, second);
```
Let's parametrize the returns type